packages/discord.js/typings/index.d.ts

```diff
@@ -2031,19 +1987,6 @@ export class InteractionCollector<Interaction extends CollectedInteraction> exte
   public collect(interaction: Interaction): Snowflake;
   public empty(): void;
   public dispose(interaction: Interaction): Snowflake;
-  public on(event: 'collect' | 'dispose' | 'ignore', listener: (interaction: Interaction) => void): this;
```

Comment:
This did set the emitted variables correctly although the `Extras` type parameter in the extends was wrong. There are no extras emitted from this class, they typings were always wrong it seems. Need to remove the `[Collection<Snowflake, Interaction>]` on the extends of this class to be correct again.